libs/zopengl/src/zopengl.zig

```diff
@@ -11,7 +11,7 @@ comptime {
 pub const bindings = @import("bindings.zig");
 pub const wrapper = @import("wrapper.zig").Wrap(bindings);
 
-pub const LoaderFn = *const fn ([:0]const u8) ?*const anyopaque;
+pub const LoaderFn = *const fn ([*:0]const u8) callconv(.C) ?*const fn () callconv(.C) void;
```

Comment:
Why callconv(.C) ?*const fn () callconv(.C) void;`? This is not correct for many gl procs.